about.html

```diff
@@ -0,0 +1,89 @@
+      <nav class="navigation">
+        <a href="" class="my-menu" title="menu bar"></a>
+        <ul class="desktop-nav">
+          <li class="nav-men">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-men">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+        </ul>
+        <div id="mobilenav" class="mobileNavigation">
+          <a href="javascript:void(0)" class="killnav" onclick="menukill()">&times;</a>
+          <a href="index.html" class="close-menu">Home</a>
+          <a href="about.html" class="close-menu">About</a>
+        </div>
+        <span class="navonmo" id="menu-toggle"><img src="./images/Union (2).png" alt="menu nav bar" style="width: 40px; height: 40px";></span>
+      </nav>
+    </header>
+    <div class="landing-page-div">
+      <h2 class="homepage-h2">"Hello! Sharing world"</h2>
+      <h1 class="homepage-h1">Revolutionizing Tutoring Through Technology Summit 2023  <img class="lineonabt" src="images/line.png" alt="line"></h1>
+      <h1 class="homepage-h8">Revolutionizing Tutoring Through <br> Technology Summit 2023  <img class="lineonabt" src="images/line.png" alt="line"></h1>
+      <p class="homepage-p">The Revolutionizing Tutoring Through Technology Summit 2023 is a premier global event that serves as a catalyst for revolutionizing the field of education through the transformative power of technology. </p>
+      <p class="homepage-p4">The Revolutionizing Tutoring Through Technology Summit 2023 is a groundbreaking event at the forefront of educational innovation and advancement. As the leading global platform for educators, technologists, policymakers, and industry leaders, the summit aims to explore and promote the transformative potential of technology in revolutionizing tutoring methods and enhancing learning outcomes.</p>
+      <p class="homepage-p2">Please contact us per Email for any futher questions about the summit</p>
+      <p class="homepage-p3">wrongemail@summit.org</p>
+    </div>
+  </section>
+
+  <section class="logo-section">
+    <div class="landing-page-div">
+      <h2 class="logo-h2">MM Global Global Summit 2015 Logo <img src="images/line.png" alt="line"></h2>
+      <p class="logo-p2">Please contact us per Email for any futher questions about the summit</p>
+      <p class="logo-p3">The logo of MM Global Summit 2023 was decided through the logo competition from 8. June to 7. july. After online public vottingand selection committees judging process, the loho from Mershark won the prize. After consulting with Mr. Mershark we optimized the price winning logo for this years summit as following </p>
+      <p class="logo-p"><img src="images/logo.png" alt="logo" width="60%" height="7%"></p>
+    </div>
+  </section>
+
+  <section class="past-summit-section">
+    <div class="landing-page-dive">
+      <h2 class="logo-h2">See the past Summit <img src="images/line.png" alt="line"></h2>
```

Comment:
> - [ ] I kindly suggest you use CSS background-image or a border line to implement the line since the image is not part of the content rather than using an `img` tag. This is for aesthetic reasons. You can [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.👍 Also ensure the size are the same as the provided design.☺